src/who.ts

```diff
@@ -0,0 +1,96 @@
+const completionSpec: Fig.Spec = {
+  name: "who",
+  description: "Display who is logged in",
+  parserDirectives: {
+    optionsMustPrecedeArguments: true,
+  },
+  subcommands: [
+    {
+      name: "am",
+      insertValue: "am I",
+      displayName: "am I",
+      subcommands: [{ name: "I" }],
+      priority: 40,
```

Comment:
Just adds descriptions
```suggestion
      description: "Returns the invoker's real user name",
      subcommands: [{
      	name: "I",
        description: "Returns the invoker's real user name",
      }],
      priority: 40,
```

Reply:
Thanks for the help through this @SeparateRecords !